wit/wit.py

```diff
@@ -69,6 +69,151 @@ def message(self, msg, context=None, n=None, verbose=None):
             params['context'] = json.dumps(context)
         resp = req(self.logger, self.access_token, 'GET', '/message', params)
         return resp
+    def new_samples(self, text,entities,*args):
+        """  Validate samples (sentence + entities annotations) to train your app programmatically.
```

Comment:
new line after """
Just link to the http doc as a single source of truth: https://wit.ai/docs/http (remove all docstring until the list of params.